src/main/java/seedu/taassist/model/ModelManager.java

```diff
@@ -19,11 +20,15 @@
  * Represents the in-memory model of TA-Assist data.
  */
 public class ModelManager implements Model {
+    private static final String DEFAULT_FOCUS_LABEL = "";
```

Comment:
I think we can see how "" looks like. If it looks like something is missing we can change it to "-" afterwards.

Reply:
@Bubbl3T This is not the content inside "[]". By "Focus Label" I am referring to the entire focus label: "Focus: [ClassName]". So making it "" removes the whole thing. Which is the default when not in focus.